src/pages/schedule-interview/graphql/mutation/cancelCandidateInterview.gql.ts

```diff
@@ -0,0 +1,15 @@
+import { graphql } from "@/__generated__"
+
+export const LICancelCandidateInterview = graphql(`
+  mutation LICancelCandidateInterview($input: LICancelCandidateInterviewDto!) {
```

Comment:
It is ok for now, but what we do right now is define the variable explicitly if it contains just one or two
```suggestion
  mutation LICancelCandidateInterview($shortcode: String!) {
```

Reply:
got it. But I think using object is convenient for later extension